templates/flavors/nvidia-gpu/gpu-operator-components.yaml

```diff
@@ -344,7 +344,7 @@ spec:
         - name: node-feature-discovery-master
           securityContext:
             {}
-          image: "quay.io/kubernetes_incubator/node-feature-discovery:v0.6.0"
+          image: "k8s.gcr.io/nfd/node-feature-discovery:v0.9.0"
```

Comment:
this is a good example of one we could switch to using the remote spec directly in the future #1917